frontends/common/parser_options.h

```diff
@@ -32,6 +32,7 @@ limitations under the License.
 // Standard include paths for .p4 header files. The values are determined by
 // `configure`.
 extern const char *p4includePath;
+extern const char *p4includeInternalPath;
```

Comment:
The discussion in #4742 makes me thing we just should have this be a set. Do not know how difficult this is to implement with all the back ends.